.travis.yml

```diff
@@ -20,12 +20,15 @@ jobs:
         - npm run danger
         - npm run lint
         - npm run jest
+        - npm run unit
         # These can't be built in parallel because the cache is a single object and you can't add to it in parallel
         - npm run compile:prod -- --env.disableAggregateThemes --output-path $TRAVIS_BUILD_DIR/travis-build/default
         - npm run compile:lowlight -- --output-path $TRAVIS_BUILD_DIR/travis-build/lowlight
         - npm run compile:fusion -- --output-path $TRAVIS_BUILD_DIR/travis-build/fusion
+    - stage: build
```

Comment:
For the build stage to run, you'll wana add to the stages config below; https://github.com/cerner/terra-graphs/pull/4/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485R169